### ISSUE TITLE
Current state of the project

### ISSUE THREAD

**Issue by aseltmann** [Git-labbook/git-labbook#2]
Hi there! I really like your [paper](https://hal.inria.fr/hal-01112795/document) and started setting up a similar workflow for my own research. I just wanted to check how you see the current state of this project. Do you use it regularly? Or did you maybe even abandon it? Are there any plans on completing the TODO lists mentioned?
All the best from Jena, Germany!

**Comment by stanisic** [comment]
Hi, and thanks for the interest! As far as I know, there havent been any recent developments regarding this project and I dont think that many people are currently using the tool. However, the principles described in the paper (and prototyped by git-labbook tool) are still valid, and the guidelines are definitely followed on a daily basis by us and many other researchers. Therefore, my view of this project is that it should serve as an example and inspire others on how they should organize their studies, probably with some domain-specific adjustments.

If you are interested in knowing more details about this work, you can check the presentation we gave at the reproducible research webinar:
https://github.com/alegrand/RR_webinars/blob/master/4_logging_and_backing_up_your_work/index.org
http://newstream.u-ga.fr/2016-06-07_Reproducible-Research_Danjean-Legrand-Stasinic.mp4
but also be free to ask us directly any questions that might arise.

Best, Luka

**Comment by aseltmann** [comment]
Hi Luka,
thank you for the fast reply. Your presentation and recording look quite helpful!
I think it is sad that the project seems to "die" - but I see the difficulties. If I speak with my colleagues from the lab who are not experienced with editors or a CLI, the steep learning curve of emacs or even the CLI would be a huge problem for adaptation, even if the need for reproducible workflows is there, especially with these basic, powerful tools. For them, a GUI approach would be the thing needed. I see you used SmartGit - maybe this is a first approach. For org-mode, this is much more difficult.<sup>1</sup>
All the best,
Alex

<sup>1</sup>possible approaches I would see: 
- editing in a browser with live preview, like in this project: http://mooz.github.io/org-js/
- preconfigured Emacs flavours like [Scimax](https://github.com/jkitchin/scimax) even though you still have the steep learning curve of learning the "Emacs flow"

**Comment by stanisic** [comment]
I agree with you and thanks for the links. Good luck with setting your research workflows!
Best, Luka
